File: prob_69a_a_young_physicist/p69a.py
def main():
    lines = int(input().strip())
    a = b = c = 0

    for i in range(lines):
        values = input().strip().split(" ")
        a += int(values[0])
        b += int(values[1])
        c += int(values[2])

    if a == 0 and b == 0 and c == 0: print("YES");
    else: print("NO");


if __name__ == "__main__":
    main();

File: prob_69a_a_young_physicist/p69a.js
"use strict";

const main = () => {
  const lines = +readline().trim();
  let a = 0,
    b = 0,
    c = 0;

  for (let i = 0; i < lines; i++) {
    const values = readline().trim().split(" ");
    a += +values[0];
    b += +values[1];
    c += +values[2];
  }

  if (a === 0 && b === 0 && c === 0) print("YES");
  else return print("NO");
};

main();
